types/oracledb/index.d.ts

```diff
@@ -2073,6 +2126,103 @@ declare namespace OracleDB {
          * @since 1.11
          */
         poolAlias?: string | undefined;
+        /**
+         * The directory in which the Optional Oracle Net Configuration Files are found.
+         * For node-oracledb Thick mode, use the oracledb.initOracleClient() option configDir.
+         * @since 6.0
+         */
+        configDir?: string | undefined;
+        /**
+         * Enables network routing through multiple protocol addresses. The value of this property can be ON or OFF.
+         * The default value is ON.
+         * For node-oracledb Thick mode, use an Easy Connect string or a Connect Descriptor string.
+         * @since 6.0
+         */
+        sourceRoute?: string | undefined;
+        /**
+         * The distinguished name (DN) that should be matched with the server. If specified, this value is used for any verification. Otherwise, the hostname will be used.
+         * This value is ignored if the sslServerDNMatch property is not set to the value True.
+         * For node-oracledb Thick mode, use an Easy Connect string or a Connect Descriptor string.
+         * @since 6.0     
+         */
+        sslServerCertDN?: string | undefined;
+        /**
+         * Determines whether the server certificate DN should be matched in addition to the regular certificate verification that is performed.
+         * If the sslServerCertDN parameter is not provided, host name matching is performed instead.
+         * The default value is True.
+         * For node-oracledb Thick mode, use an Easy Connect string or a Connect Descriptor string.
+         * @since 6.0
+         */
+        sslServerDNMatch?: boolean | undefined;
+        /**
+         * The name or IP address of a proxy host to use for tunneling secure connections.
+         * For node-oracledb Thick mode, use an Easy Connect string or a Connect Descriptor string.
+         */
+        httpsProxy?: string | undefined;
+        /**
+         * 	
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/oracledb/index.d.ts;
  Line: 2163;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.